docs/configurations.md

```diff
@@ -236,14 +290,26 @@ scalar.db.password=<PASSWORD>
 
 ### Example 2
```

Comment:
Since this example is for ScalarDB Server, I think we should remove it from this doc in `master`, `3.10`, and `3.9` branches. 

Is there another ScalarDB example configuration that we can provide here for the `master`, `3.10`, and `3.9` branches?